google-cloud-bigtable/src/main/resources/changestream.json

```diff
@@ -0,0 +1,1232 @@
+            "tiebreaker": 100,
+            "token": "data-change-token",
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ]
+          }
+        }
+      ],
+      "results": [
+        {
+          "error": true
+        }
+      ]
+    },
+    {
+      "description": "GC mutation no source cluster id",
+      "api_responses": [
+        {
+          "data_change": {
+            "row_key": "0000000000000000",
+            "type": "GARBAGE_COLLECTION",
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "token": "data-change-token",
+            "low_watermark": "2022-07-01T00:00:00Z",
+            "chunks": [
+              {
+                "mutation": {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              }
+            ],
+            "done": true
+          }
+        }
+      ],
+      "results": [
+        {
+          "record": {
+            "change_stream_mutation": {
+              "row_key": "0000000000000000",
+              "type": "GARBAGE_COLLECTION",
+              "commit_timestamp": "2022-07-01T00:00:00Z",
+              "tiebreaker": 100,
+              "token": "data-change-token",
+              "low_watermark": "2022-07-01T00:00:00Z",
+              "mutations": [
+                {
+                  "delete_from_family": {
+                    "family_name": "family"
+                  }
+                }
+              ]
+            }
+          },
+          "error": false
+        }
+      ]
+    },
+    {
+      "description": "1 chunked SetCell([{SC_chunk1}, {SC_chunk2}]->ChangeStreamMutation{SC})",
+      "api_responses": [
+        {
+          "data_change": {
+            "row_key": "0000000000000000",
+            "type": "USER",
+            "source_cluster_id": "source-cluster-id",
+            "commit_timestamp": "2022-07-01T00:00:00Z",
+            "tiebreaker": 100,
+            "chunks": [
+              {
+                "chunk_info": {
+                  "chunked_value_size": 1
+                },
+                "mutation": {
+                  "set_cell": {
+                    "family_name": "family",
+                    "column_qualifier": "0000000000000000",
+                    "timestamp_micros": 1000,
+                    "value": "dg=="
+                  }
+                }
+              }
+            ]
+          }
+        },
+        {
+          "data_change": {
+            "type": "CONTINUATION",
+            "token": "data-change-token",
+            "low_watermark": "2022-07-01T00:00:00Z",
+            "chunks": [
+              {
+                "chunk_info": {
+                  "chunked_value_offset": 1,
+                  "chunked_value_size": 1,
```

Comment:
is the chunked value size the total size of 2 chunks?

Reply:
Maybe update this to chunked_value_size: 2 and set a value in the mutation as well so you could verify this logic: https://github.com/tengzhonger/java-bigtable/blob/7e763c66a6e6d346e503f81ceaad0f3a8ba92f1a/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/changestream/ChangeStreamStateMachine.java#L467

Reply:
Good point! Updated the tests.